minoc/src/main/java/org/opencadc/minoc/PutAction.java

```diff
@@ -136,37 +136,30 @@ public Content accept(String name, String contentType, InputStream inputStream)
     public void doAction() throws Exception {
         
         initAndAuthorize(WriteGrant.class);
-        
-        String md5Header = syncInput.getHeader("Content-MD5");
+
+        String digestHeader = syncInput.getHeader("Digest");
```

Comment:
SyncOutput.setDigest(URI) exists... why not add `URI SyncInput.getDigest()` to match what clients have in `HttpTransfer`? Should not be using DigestUtil here

Reply:
Agree, added URI getDigest() to SyncInput.